server.js

```diff
@@ -1,10 +1,12 @@
 var serverPort = +process.env.PORT || +process.argv[2] || 80;
+var bindAddress = process.env.BIND_ADDRESS || process.argv[3] || '127.0.0.1';
```

Comment:
I switched the default back because, for reasons I haven't yet researched fully, Heroku doesn't work with this default.

624f93f